setup.py

```diff
@@ -247,6 +247,10 @@ def libadcc_extension():
     define_macros = []
     search_system = True
 
+    if sys.platform == "darwin" and is_conda_build():
```

Comment:
Are you sure this is the "right" thing to test for. Isn't this rather a clang-specific thing?

Reply:
Kind of... AppleClang does not complain, and the only macOS compiler on `conda` is clang... I can add a note though.

Reply:
But what if I use clang on ubuntu?

Reply:
Yes... so we need to get the compiler name and check for that 😄

Reply:
Mhm do you know by heart how to get the compiler name outside the extension class? 🤕

Reply:
Otherwise, I think one can just add the two flags directly to `extra_compile_args` without a compiler/platform distinction.

Reply:
> how to get the compiler name outside the extension class?

No, but I guess you can always get it *after* the extension has been created (then the user can't override it, but at least things compile.

I think in order to decide what to do you will need to test if this problem also occurs in other circumstances. If it's conda-specific I don't mind the way you solved it now. If it's not then I think it should be solved more generally.

(On that note I think the siteconfig solution is not the best way to tackle it, better disable it in the setup.py)

Reply:
It seems to be conda+darwin-specific. Clang on arch compiles without complaining.

Reply:
Ok cool. Then I'm happy as it is.

Reply:
Giving conda another run with the current configuration... Then I can also run some timings using the conda builds 😄